apps/microsoft-teams/backend/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@contentful/microsoft-teams-app",
```

Comment:
small nit: maybe call this `@contentful/microsoft-teams-backend` to mirror what we have in the frontend. Then the  top level `package.json` could be named `@contentful/microsoft-teams-app` or `@contentful/microsoft-teams`